graphql_compiler/__init__.py

```diff
@@ -140,7 +140,11 @@ def get_graphql_schema_from_orientdb_schema_data(schema_data, class_to_field_typ
     """Construct a GraphQL schema from an OrientDB schema.
 
     Args:
-        schema_data: list of dicts describing the classes in the OrientDB schema.
+        schema_data: list of dicts describing the classes in the OrientDB schema. The following
+                     format is the way the data is structured in OrientDB 2. See
+                     test_get_graphql_schema_from_orientdb_schema in
+                     graphql_compiler/tests/integration_tests/test_backends_integration.py
+                     for an example of how to query this data.
```

Comment:
`example of how to get this data from orientdb`. Querying is easy, it's a dict :D